docs/root/version_history/current.rst

```diff
@@ -26,6 +26,7 @@ Minor Behavior Changes
   in the environment.
 * router: added transport failure reason to response body when upstream reset happens. After this change, the response body will be of the form `upstream connect error or disconnect/reset before headers. reset reason:{}, transport failure reason:{}`.This behavior may be reverted by setting runtime feature `envoy.reloadable_features.http_transport_failure_reason_in_body` to false.
 * router: now consumes all retry related headers to prevent them from being propagated to the upstream. This behavior may be reverted by setting runtime feature `envoy.reloadable_features.consume_all_retry_headers` to false.
+* runtime: add envoy.memory.heap_shrink_threshold run time key to tune the heap shrink.
```

Comment:
ref link to the heap shrink section you added

Reply:
done